docs/community/channels.rst

```diff
@@ -14,8 +14,8 @@ Chatrooms
 ---------
 Most day-to-day communication is happening in our chatrooms:
 
-- `Public chatroom <https://gitter.im/inveniosoftware/invenio>`_ (for everyone)
-- `Developer chatroom <https://gitter.im/inveniosoftware/invenio>`_ (for
+- `Public chatroom <https://discord.gg/8qatqBC>`_ (for everyone)
+- `Developer chatroom <https://discord.gg/8qatqBC>`_ (for
```

Comment:
Weird two have the same link twice (though it was already this way).

Reply:
Let's list only one link then. Later on, once Discord evolves and new chats are created, we can extend this section:
```diff
+ - `Public chatroom <https://discord.gg/8qatqBC>`_ (for everyone)
- - `Developer chatroom <https://discord.gg/8qatqBC>`_ (for
```

Could you also rebase against latest master?